src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
What is using this import in this file?

Reply:
So from my memory, the placeholder NavBar needs the CSS in order to render properly, but since we're using Yi's now, this can probably be removed. I can double-check on this though.